python/mxnet/ndarray/sparse.py

```diff
@@ -327,7 +327,11 @@ def __getitem__(self, key):
         array([[0, 0, 3]], dtype=float32)
         """
         if isinstance(key, int):
-            raise ValueError("__getitem__ with int key is not implemented for CSRNDArray")
```

Comment:
could you update the documentation with `key: int or slice` and add the example accordingly? Also we should add unit test for this

Reply:
Done.